scripts/run_integration_tests_aarch64.sh

```diff
@@ -11,12 +11,13 @@ build_spdk_nvme() {
     SPDK_DIR="$WORKLOADS_DIR/spdk"
     SPDK_REPO="https://github.com/spdk/spdk.git"
     SPDK_DEPLOY_DIR="/usr/local/bin/spdk-nvme"
-    checkout_repo "$SPDK_DIR" "$SPDK_REPO" master "6301f8915de32baed10dba1eebed556a6749211a"
+    checkout_repo "$SPDK_DIR" "$SPDK_REPO" master "ef8bcce58f3f02b79c0619a297e4f17e81e62b24"
```

Comment:
Please don't use a different version to x86-64.

Reply:
To make it clear, you will need to bump the spdk version for both x86 and aarch64 at the same time. You can do it for x86_64 from https://github.com/cloud-hypervisor/cloud-hypervisor/blob/main/resources/Dockerfile#L133

Reply:
Hi @rbradford @likebreath ,The new code verified.